packages/primitives/package.json

```diff
@@ -30,13 +30,13 @@
     "blob-polyfill": "7.0.20220408",
     "esbuild-plugin-alias": "latest",
     "event-target-shim": "6.0.2",
-    "formdata-node": "4.4.1",
+    "formdata-node": "5.0.0",
```

Comment:
v5 is dropping node12 and use ESM; we have to be sure ESM is converted to CommonJS by our TS build.

Reply:
tested via REPL:

```
~/vercel/edge-runtime/packages/runtime primitives ≡
❯ node dist/cli/repl.js
ƒ => const formData = new FormData();
undefined
ƒ => formData
FormData { [Symbol(state)]: [] }
ƒ => formData.append('foo', 'bar')
undefined
ƒ => formData
FormData { [Symbol(state)]: [ { name: 'foo', value: 'bar' } ] }
ƒ =>
```